nimbus/src/config.rs

```diff
@@ -14,6 +14,6 @@
 /// - `bucket_name`: The name of the bucket containing the collection on the server
 #[derive(Debug, Clone)]
 pub struct RemoteSettingsConfig {
-    pub server_url: Option<String>,
+    pub server_url: String,
```

Comment:
While we are at it, is there any reason not to include the `collection_name` within this struct?